crates/wasmtime/src/runtime/vm/stack_switching.rs

```diff
@@ -39,15 +39,11 @@ pub use stack::*;
 /// For performance reasons, the VMContRef at the bottom of this chain
 /// (i.e., the one pointed to by the VMContObj) has a pointer to the
 /// other end of the chain (i.e., its last ancestor).
-// FIXME(frank-emrich) Does this actually need to be 16-byte aligned any
-// more? Now that we use I128 on the Cranelift side (see
-// [wasmtime_cranelift::stack_switching::fatpointer::pointer_type]), it
-// should be fine to use the natural alignment of the type.
 #[repr(C, align(16))]
```

Comment:
I don't think this align(16) needs to be here anymore, as described in the deleted `FIXME` comment? But if we are leaving it, then we should have a follow up item to investigate in the meta issue and also should adjust the `size_of_vmcontobj` method above to align to 16 instead of the natural alignment that my suggestion comment sketched out.